### PR TITLE
Add a basic status container

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- pass the status flag with guiproxy
- you should see a "status" button next to apps/machines
- clicking it should reveal the spot that the status UI will appear

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7929/

File: jujugui/static/gui/src/app/app.js [hunk 1]
```diff
@@ -1195,7 +1195,8 @@ YUI.add('juju-gui', function(Y) {
           appState={this.state}
```
Comment (frankban) [inline jujugui/static/gui/src/app/app.js]: All good: Jeff will ask to port the same changes to the new init.js and init/... renderer ;-)

File: jujugui/static/gui/src/app/components/status/status.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        </div>
+      </juju.components.Panel>
+    );
+  }
+
+};
+
+
+Status.propTypes = {
+  acl: PropTypes.object,
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/status/status.js]: Are we sure we need acl?

File: jujugui/static/gui/src/app/components/status/test-status.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+describe('Status', function() {
+  beforeAll(function(done) {
+    // By loading this file it adds the component to the juju components.
+    YUI().use('status', function() {
+      done();
+    });
+  });
+
+  it('can render', () => {
+    const renderer = jsTestUtils.shallowRender(
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/status/test-status.js]: I'd abstract out a render(props) method that can be reused by all tests that will be implemented here.

File: jujugui/static/gui/src/app/components/status/status.js (hunk 2 above)
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/status/status.js]: Good point, probably not.

File: jujugui/static/gui/src/app/app.js (hunk 1 above)
Comment (huwshimi) [inline jujugui/static/gui/src/app/app.js]: Fair call :)

File: jujugui/static/gui/src/app/components/status/test-status.js (hunk 3 above)
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/status/test-status.js]: Do you think that's necessary for a component with one prop?

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7930/

**Comment by frankban** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**